docs/src/modules/sandbox/CodeSandbox.ts

```diff
@@ -35,7 +35,29 @@ function openSandbox({ files, codeVariant, initialFile }: any) {
 
 function createReactApp(demoData: DemoData) {
   const ext = getFileExtension(demoData.codeVariant);
-  const { title, githubLocation: description } = demoData;
+  const { title, relativeModules = [], githubLocation: description } = demoData;
+
+  if (relativeModules) {
```

Comment:
Can you extract this in a separate function and re-use it in both places?

Reply:
Yes; for context why I decided to not do this in the first place is because I anticipated having to tweak the logic a bit given the nuances for each platform